### DISCUSSION TITLE
Just a simple thank you

### DISCUSSION THREAD

**Discussion by DonMischo** [adieyal/sd-dynamic-prompts#674] [Show and tell]
Thanks a lot for this awesome tool. I'm using it a lot. :)

You've collected a lot of stuff related to your extension, too, including guides. I happy that my little guide on civitai made it into this list, too.

I'm continously improving my Billions of Wildcards Set, maybe you could add it to the list of wildcard packs, too. ;)
https://civitai.com/models/138970/billions-of-wildcards-all-in-one

Keeping going <3

**Comment by adieyal** [comment]
Thanks @DonMischo - Appreciation is much appreciated. 

I've added Billions of Wildcards to the [list](https://github.com/adieyal/sd-dynamic-prompts/blob/main/docs/resources.md#wildcard-packs) - There have been a number of mentions of it recently so clearly it is gaining in popularity. I scanned through it, looks like a labour of love. 

In case you haven't looked at it, you might be interested in this file as an example of creating re-usable prompt templates in yaml - https://github.com/adieyal/sd-dynamic-prompts/blob/main/collections/publicprompts.yaml

**Reply by DonMischo** [reply]
Thx I'll check it out. It is definitly a labour of love. :) I really enjoy the images they can make. Wouldn't be possible without this extension :D

**Reply by DonMischo** [reply]
I think I'm using those already ;) I called them "easy to use" wildcards, which use many of the subsets to create versatile chars, buildings, clothings etc. :)

**Comment by timwillhack** [comment]
I didn't want to add another one of these to your discussion list, but boy oh boy is this extension magical.  I wrote my own similar thing for generating prompts for story generation for text generation but this is better than what I made and I love it.  Keep up the great work!
